test/integration/members.test.js

```diff
@@ -38,41 +38,35 @@ describe("Members", function () {
   let jwt;
 
   afterEach(async function () {
-    await addUser();
-  });
-
-  after(async function () {
     await cleanDb();
   });
 
   describe("GET /members", function () {
-    before(async function () {
-      await cleanDb();
-    });
-
-    afterEach(async function () {
+    beforeEach(async function () {
       await addUser(userWithoutRolesObject);
       await addUser(userWithRolesObjectWithoutArchivedProperty);
       await addUser(userWithRolesObjectWithArchivedTrue);
       await addUser(userWithRolesObjectWithArchivedFalse);
     });
 
     it("Should return empty array if no user is found", function (done) {
-      chai
-        .request(app)
-        .get("/members")
-        .end((err, res) => {
-          if (err) {
-            return done(err);
-          }
+      cleanDb().then(() => {
```

Comment:
Is after each doesn't taking care of this ??

Reply:
After each cleans the DB but then beforeEach adds the users in the DB.

Reply:
Continuing the conversation from here as its the same comment https://github.com/Real-Dev-Squad/website-backend/pull/681#discussion_r945517685